src/Sarif/PropertiesDictionary.cs

```diff
@@ -37,11 +36,6 @@ public PropertiesDictionary() : this(null)
         {
         }
 
-        protected PropertiesDictionary(SerializationInfo info, StreamingContext context)
-            : base(info, context)
-        {
-        }
```

Comment:
check if the serialization will be the same comparing previous vs current

Reply:
got it, thanks

Reply:
the file written are the same